examples/go.mod

```diff
@@ -59,10 +59,11 @@ require (
 	github.com/libp2p/go-reuseport v0.2.0 // indirect
 	github.com/libp2p/go-yamux/v4 v4.0.0 // indirect
 	github.com/libp2p/zeroconf/v2 v2.2.0 // indirect
-	github.com/lucas-clemente/quic-go v0.31.0 // indirect
+	github.com/lucas-clemente/quic-go v0.31.1 // indirect
```

Comment:
There's still a dependency upgrade here.

Reply:
@marten-seemann  fixed

Reply:
Well, this one. There's another one.